src/utils/getParentChainFromId.ts

```diff
@@ -0,0 +1,15 @@
+import { extractChain } from 'viem';
+import { ParentChain, isValidParentChainId } from '../types/ParentChain';
+import { chains } from '../chains';
+
+export function getParentChainFromId(chainId: number): ParentChain {
```

Comment:
Useful function to allow users to specify the parent chain id in the .env file of the examples, and obtain the Viem chain configuration from there (instead of having to change the imported chain in the index.ts file of every example).